config/trustedproxy.php

```diff
@@ -23,7 +23,7 @@
      * of your proxy (e.g. if using ELB or similar).
      *
      */
-    'proxies' => \LibreNMS\Util\Env::parseArray('APP_TRUSTED_PROXIES', '*', ['', '*', '**']),
+    'proxies' => \LibreNMS\Util\Env::parseArray('APP_TRUSTED_PROXIES', null, ['', '*', '**']),
```

Comment:
@librenms/reviewers this could be a potential breaker for some installations, anyone uses a proxy that can test?  
`null` is default and more secure, shouldn't blindly trust all proxies by default; https://laravel.com/docs/7.x/requests#configuring-trusted-proxies

Reply:
Best to change it to
```php
'proxies' => \LibreNMS\Util\Env::parseArray('APP_TRUSTED_PROXIES', ['*']),
```

I'm fine with changing the default, but it would be best to do it in a separate PR and send notifications prior.

Reply:
Reverted to upstream as it didn't work with any of the above.

I never got an error, what were you running into?

Reply:
failed when running phpunit, but lets do a seperate pr

Reply:
I was able to reproduce the issue.  It was a little quirky.

Problem was with the "dummy" request in Larave::bootWeb()